raiden/api/python.py

```diff
@@ -1733,8 +1732,8 @@ def create_light_client_payment(
                 privkey=self.raiden.privkey,
             )
             if prev_secrethash:
-                current_payment_task = chain_state.payment_mapping.secrethashes_to_task[prev_secrethash]
-                chain_state.payment_mapping.secrethashes_to_task.update(
+                current_payment_task = chain_state.payment_mapping[creator_address].secrethashes_to_task[prev_secrethash]
+                chain_state.payment_mapping[creator_address].secrethashes_to_task.update(
```

Comment:
these seem long. maybe it would be useful to define getter-type functions on the chain state or the payment mapping types to make this less verbose.